### PR TITLE
Fix outdated GraphicsGems link

### PR DESCRIPTION
**Pull request by ahojnnes**

### CODE REVIEW

File: skimage/exposure/_adapthist.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def equalize_adapthist(image, ntiles_x=None, ntiles_y=None, clip_limit=0.01,
 
     References
     ----------
-    .. [1] http://tog.acm.org/resources/GraphicsGems/gems.html#gemsvi
+    .. [1] http://tog.acm.org/resources/GraphicsGems/
```
Comment (soupault) [inline skimage/exposure/_adapthist.py]: Why not `http://www.realtimerendering.com/resources/GraphicsGems/gems.html#gemsiv`?

Comment (soupault) [inline skimage/exposure/_adapthist.py]: I believe it is just a typo.
Also, there is one broken link in the header - https://github.com/ahojnnes/scikit-image/blob/7353785603eebe1f37bb98bdc44326e3e5f19707/skimage/exposure/_adapthist.py#L5 .

**Comment by ahojnnes** [comment]
@soupault Should be done.

**Comment by soupault** [comment]
@ahojnnes I'd personally keep for the `References` a link to the detailed list of contents (i.e. `gems.html`), so user can easily (`C-f` -> `CLAHE`) find the reference code.

Not going to argue about this, though. LGTM :+1:

**Comment by ahojnnes** [comment]
@soupault The reason I am changing this is that I got an email from one of their people asking me to update the link to the one I committed.

**Pull request merged**